ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -613,6 +616,19 @@ public class RouterConfig {
   public final int routerMaxNumMetadataCacheEntries;
   public static final int MAX_NUM_METADATA_CACHE_ENTRIES_DEFAULT = 10;
 
+  /**
+   * Blobs created after this timestamp can rely on metadata chunk to get the overall status of update operations
+   * (ttl update, undelete) on the blob.
+   * This config is expected to be a temporary stop gap to improve the resiliency of update operations on a blob, until
+   * the server version changes are deployed to identify update operations that can rely on metadata chunk for the
+   * update status of the entire blob.
+   * The value of this config will be an upper bound of the timestamp when the frontend code version that updates
+   * metadata chunk last has been deployed to all frontend hosts on a cluster.
+   * A value of LONG.MAX_VALUE for this config would mean that this config is effectively disabled.
+   */
+  @Config(ROUTER_UPDATE_OP_METADATA_RELIANCE_TIMESTAMP_IN_MS)
+  public final long routerUpdateOpMetadataRelianceTimestampInMs;
```

Comment:
instead of a timestamp, can we bump up the metadata version of the metadata chunk if there is one ?

Reply:
discussed offline. this is a stop-gap until we can complete the final solution.